plugins/woocommerce/src/Blocks/Templates/AbstractTemplate.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\Templates;
+
+use Automattic\WooCommerce\Blocks\BlockTemplatesRegistry;
+use Automattic\WooCommerce\Blocks\Utils\BlockTemplateUtils;
+
+/**
+ * AbstractTemplate class.
+ *
+ * Shared logic for templates.
+ *
+ * @internal
+ */
+abstract class AbstractTemplate {
+
+	/**
+	 * The slug of the template.
+	 *
+	 * @var string
+	 */
+	const SLUG = '';
```

Comment:
I see we have used `const SLUG` in this class and `public $slug` in the `AbstractTemplatePart` class. Is there a reason for that?

Reply:
Ups, no. That was a typo. Good catch! It should have been fixed with b53dc771d652578d2f99722f28e407b7b3544584. :slightly_smiling_face: